auth/routes/oauth_routes.go

```diff
@@ -6,10 +6,12 @@ const (
 	OAuthBegin    = "OAuthBegin"
 	OAuthCallback = "OAuthCallback"
 	LogOut        = "LogOut"
+	PostCustom    = "PostCustom"
 )
 
 var OAuthRoutes = rata.Routes{
 	{Path: "/auth/logout", Method: "GET", Name: LogOut},
 	{Path: "/auth/:provider/callback", Method: "GET", Name: OAuthCallback},
 	{Path: "/auth/:provider", Method: "GET", Name: OAuthBegin},
+	{Path: "/auth/:provider/", Method: "POST", Name: PostCustom},
```

Comment:
The routes can be found here: https://github.com/Chumper/atc/blob/4fb3dd3c9beba37e46a32967d7843b649b130077/auth/github/routes.go

We add a `/token` endpoint to return a bearer token when a valid personal auth token is provided.